tests/test-cases/configloaders/qube_validation_error.json

```diff
@@ -0,0 +1,25 @@
+{
+    "$schema": "https://purl.org/csv-cubed/qube-config/v1",
+    "id": "ons-international-trade-in-services-by-subnational-areas-of-the-uk",
+    "title": "International trade in services by subnational areas of the UK",
+    "description": "Experimental estimated value of exports and imports of services for 2018 for NUTS1, NUTS2, NUTS3 and 15 city regions, including industry and non-EU and EU split.",
+    "landingPage": "https://www.ons.gov.uk/businessindustryandtrade/internationaltrade/datasets/internationaltradeinservicesbysubnationalareasoftheuk",
+    "publisher": "https://example.com",
+    "published": "2020-09-10",
+    "families": [
```

Comment:
This configuration is from an info.json style file. Is there a reason this is in a qube-config.json file?

Reply:
gonna tag in @canwaf , I only really got behave working

Reply:
think I follow this, we're testing the JsonSchema validation here, if I remove $schema or unit, or measure or any of the additional things then we fail the "cli will succeed" step and don't get to the schema validation that we're after.

Reply:
JSON Schema validation never causes the CLI to fail. It just provides warnings to the user (which we call schema validation errors)

Reply:
yeah I got that, turned out it was all because we had a step `Then the csvcubed CLI should succeed` which it didn't without the extra info (we'd get a failed status code from say a non schema based "must have a unit" validation check).

removing that step let me strip down the input, I think its more like what you had in mind now.